### ISSUE TITLE
Show Movies/Shows That Are Already In Library

### ISSUE THREAD

**Issue by rrbailey89** [tidusjar/PlexRequests.Net#539]
If this is a bug report please make sure you have filled the following in:
(If it's not a bug and a feature request then just remove the below)

#### Plex Requests.Net Version:
1.91

#### Operating System:

Windows 10
#### Applicable Logs (from `/logs/` directory or the Admin page):

```
Logs go here (Please make sure you remove any personal information from the logs)
```

#### Reproduction Steps:

I have tested this app out and it looks great! I tested it by trying to request a movie Avatar which I already have in my library of movies in plex. It allowed me to request it. Does this app know whats in the library already or have the capabilities?

**Comment by tidusjar** [comment]
It does have the capability! There is currently a bug I need to fix though! I broke it with the latest update.

**Comment by rrbailey89** [comment]
@tidusjar okay :) i see my movies showing up as "Available" in the search.. but not for TV Shows.. before nothing showed up at all.

**Comment by tidusjar** [comment]
@rrbailey89 all good in the latest update?

**Comment by rrbailey89** [comment]
@tidusjar they show available now, but Ive noticed that its not in the same location is a movie that is available. Is this by design?

**Comment by tidusjar** [comment]
What do you mean?

**Comment by rrbailey89** [comment]
@tidusjar notice the location of the word available

![](http://i.imgur.com/gducszN.png)

![](http://i.imgur.com/e2avM7q.png)

personally I like the second one more

**Comment by tidusjar** [comment]
Ah you mean the small label?

I did think about adding this to movies, but it looked strange since the button says Available.

What do you think?

**Comment by rrbailey89** [comment]
My personal opinion is that I think it would look better if it were available hide the button (or don't load it at all) that says available and use the label from TV shows instead. And if it is available just have the option to mark unavailable if for w/e reason it's actually not available. ... I hope this makes sense

**Comment by wilsonic** [comment]
On multiple occasions I have had shows requested that I already have. I think because the request button is still visible the users will not notice the available tag or the view in plex and just click request as they usually do. Maybe adding a big available button above the Request button would be better.

**Comment by tidusjar** [comment]
@wilsonic The only problem I have with that, is that I want to try and keep all of the searches consistent. Maybe I could change the button to say available but still allow them to click it, but then it's not obvious to the user.

**Comment by wilsonic** [comment]
@tidusjar Maybe an option could be to put the View in Plex button above the request at the larger size and in green so the hierarchy helps the user see that it's available. Or have a tooltip pop out when you click request saying it's already available are you sure you want to request this?

**Comment by whitecollar17** [comment]
fixed in the release of 1.9.5
